### PR TITLE
docs(feat): smtp for elastic metal MTA-1952

### PR DESCRIPTION
**Pull request by bene2k1**
### Your checklist for this pull request

- [ ] Check that the commit messages match our requested structure.
- [ ] Name your pull request according to the [contribution guidelines](../docs/CONTRIBUTING.md).

### Description
Enable SMTP for EM

### CODE REVIEW

File: compute/elastic-metal/how-to/enable-smtp.mdx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+content:
+  h1: How to enable STMP for Elastic Metal servers
+  paragraph: This page explains how to enable STMP for Elastic Metal servers
+dates:
+  validation: 2022-05-04
+  posted: 2022-05-04
+---
+
+By default, the SMTP ports (25, 465, 587) of your Elastic Metal server are blocked, and cannot be reached from our infrastructure. 
+If you want to send e-mails from your server, you can enable SMTP for each Available Zone from the console.
```
Comment (ldecarvalho-doc) [inline compute/elastic-metal/how-to/enable-smtp.mdx:14]: add link to AZ concept?
```suggestion
If you want to send e-mails from your server, you can enable SMTP for each Availability Zone from the console.
```

File: compute/elastic-metal/how-to/enable-smtp.mdx [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+- You have an account and are logged into the [Scaleway console](https://console.scaleway.com) 
+- You have [created](/compute/elastic-metal/how-to/create-server) and [installed](/compute/elastic-metal/how-to/install-server) an Elastic Metal server
+- You have [verified your identity](/console/my-account/how-to/verify-identity)
+
+</Message>
+
+<Message type="tip"> 
+
+This guide explains how to enable SMTP on Elastic Metal servers. If you want to enable SMTP on an Instance, follow our doucmentation [how to send emails from your Instance](compute/instances/how-to/send-emails-from-your-instance).
```
Comment (ldecarvalho-doc) [inline compute/elastic-metal/how-to/enable-smtp.mdx:26]:
```suggestion
This guide explains how to enable SMTP on Elastic Metal servers. If you want to enable SMTP on an Instance, follow our documentation on [how to send emails from your Instance](compute/instances/how-to/send-emails-from-your-instance).
```

File: compute/elastic-metal/how-to/enable-smtp.mdx [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+</Message>
+
+
+1. Click **Elastic Metal** in the Compute section of the side menu in the [Scaleway console](https://console.scaleway.com).
```
Comment (ldecarvalho-doc) [inline compute/elastic-metal/how-to/enable-smtp.mdx:31]:
```suggestion
1. Click **Elastic Metal** in the **Compute** section on the side menu of the [Scaleway console](https://console.scaleway.com).
```

File: compute/elastic-metal/how-to/enable-smtp.mdx [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+2. Click the **General Settings** tab. The SMTP configuration displays.
+
+    <Message type="important">
+
+    You have to [verify your identity](/console/my-account/how-to/verify-identity) before you can enable the SMTP ports.
+
+    </Message>
+
+3. Toggle the switch to **on** for each Available Zone you want to enable STMP.
```
Comment (ldecarvalho-doc) [inline compute/elastic-metal/how-to/enable-smtp.mdx:41]:
```suggestion
3. Toggle the switch to **ON** beside each Availability Zone for which you want to enable STMP.
```

File: compute/elastic-metal/how-to/enable-smtp.mdx [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+
+    <Lightbox src="scaleway-enable-smtp.webp" alt="" />
+
+    <Message type="note">
+
+    If SMTP is not available for a specific Available Zone, the zone will display as greyed-out.
```
Comment (ldecarvalho-doc) [inline compute/elastic-metal/how-to/enable-smtp.mdx:47]:
```suggestion
    If SMTP is not available for a specific Availability Zone, the zone will display as greyed-out. 
```

File: compute/elastic-metal/how-to/index.mdx [hunk 6]
```diff
@@ -1,8 +1,141 @@
...
+## Gitlab - Overview
+
+[GitLab](https://about.gitlab.com/) is an open-core Git repository manager that provides a wiki, issue tracking and CI/CD pipeline features. Open-core means that the core functionalities of the software are released under an open-source license with additional optional modules.
+Many large technology companies use this tool to manage their software development life cycle. It was initially developed in Ruby and currently also uses Go and Vue.js as part of its technology stack.
+
+<Message type="requirement">
+
+- You have an account and are logged into the [Scaleway console](https://console.scaleway.com)  
+- You have [configured your SSH Key](/console/my-project/how-to/create-ssh-key/)
+- You have an Instance or Elastic Metal server running Ubuntu Bionic Beaver (18.04)
```
Comment (ldecarvalho-doc) [inline compute/elastic-metal/how-to/index.mdx:25]:
```suggestion
- You have an Instance or an Elastic Metal server running Ubuntu Bionic Beaver (18.04)
```

File: compute/elastic-metal/how-to/index.mdx [hunk 7]
```diff
@@ -1,8 +1,141 @@
...
+- You have a domain or subdomain configured to point to the IP address of your Instance
+- You have [enabled the SMTP ports](/compute/instances/how-to/send-emails-from-your-instance/) to send emails from your Instance
+
+</Message>
+
+## Installing GitLab
+
+1. Connect to your Instance using [SSH](/compute/instances/how-to/connect-to-instance/)
```
Comment (ldecarvalho-doc) [inline compute/elastic-metal/how-to/index.mdx:33]:
```suggestion
1. Connect to your Instance using [SSH](/compute/instances/how-to/connect-to-instance/).
```

File: compute/elastic-metal/how-to/index.mdx [hunk 8]
```diff
@@ -1,8 +1,141 @@
...
+  ```
+  ssh root@<your_instance_ip>
+  ```
+
+2. Update the `apt` packet cache and upgrade the software already installed on the instance:
```
Comment (ldecarvalho-doc) [inline compute/elastic-metal/how-to/index.mdx:38]:
```suggestion
2. Update the `apt` packet cache and upgrade the software already installed on the Instance:
```

File: compute/elastic-metal/how-to/index.mdx [hunk 9]
```diff
@@ -1,8 +1,141 @@
...
+  ```
+
+5. Set your domain name in the `EXTERNAL_URL` variable, then install GitLab using `apt`:
+  ```
+  EXTERNAL_URL="https://gitlab.example.com" apt install -y gitlab-ce
+  ```
+
+  <Message type="important">
+
+  GitLab will automatically request a [Let's Encrypt](/tutorials/configure-nginx-lets-encrypt/) certifcate for your domain name. Make sure you have pointed the `A-Record` of your domain to your Instances IP address to be able to obtain a valid certificate.
```
Comment (ldecarvalho-doc) [inline compute/elastic-metal/how-to/index.mdx:60]:
```suggestion
  GitLab will automatically request a [Let's Encrypt](/tutorials/configure-nginx-lets-encrypt/) certifcate for your domain name. Make sure you have pointed the `A-Record` of your domain to your Instance's IP address to obtain a valid certificate.
```

File: compute/elastic-metal/how-to/index.mdx [hunk 10]
```diff
@@ -1,8 +1,141 @@
...
+
+## Creating a Managed Database
+
+1. [Create a Managed PostgreSQL Database Instance](/storage/database/how-to/create-a-database/) from the Scaleway console. We name our database `gitlab_production`.
+
+  You are taken to your Managed Database's **Overview** page.
+
+2. Click the **Users** tab, then on **Add User**.
+
+3. Enter the username (`gitlab`) and a secure password. In the permission tab, give `ALL` permissions for the user on the `rdb` database.
```
Comment (ldecarvalho-doc) [inline compute/elastic-metal/how-to/index.mdx:72]:
```suggestion
3. Enter the username (`gitlab`) and a secure password. In the **Permissions** tab, give `ALL` permissions to the user on the `rdb` database.
```

File: compute/elastic-metal/how-to/index.mdx [hunk 11]
```diff
@@ -1,8 +1,141 @@
...
+    # Connection details of the Scaleway Elements Database
+    gitlab_rails['db_adapter'] = 'postgresql'
+    gitlab_rails['db_encoding'] = 'utf8'
+    gitlab_rails['db_database'] = '<SCALEWAY_DATABASE_NAME>'
+    gitlab_rails['db_host'] = '<SCALEWAY_DATABASE_IP>'
+    gitlab_rails['db_port'] = '<SCALEWAY_DATABASE_PORT>'
+    gitlab_rails['db_username'] = '<SCALEWAY_DATABASE_USERNAME>'
+    gitlab_rails['db_password'] = '<SCALEWAY_DATABASE_PASSWORD>'
+    ```
+    Replace the following values in the configuration:
```
Comment (ldecarvalho-doc) [inline compute/elastic-metal/how-to/index.mdx:96]: remove "Elements" from all mentions below

File: compute/elastic-metal/how-to/index.mdx [hunk 12]
```diff
@@ -1,8 +1,141 @@
...
+    - `<SCALEWAY_DATABASE_BASE>`: The name of your Scaleway Elements Database. In our example, this is `rdb`
+    - `<SCALEWAY_DATABASE_IP>`: The IP address of your Scaleway Elements Database Instance
+    - `<SCALEWAY_DATABASE_PORT>`: The port of your Scaleway Elements Database Instance
+    - `<SCALEWAY_DATABASE_USERNAME>`: Your Scaleway Elements Database user
+    - `<SCALEWAY_DATABASE_PASSWORD>`: Password of your Scaleway Elements Database user
+
+3. Once modified save the file and exit the text editor.
```
Comment (ldecarvalho-doc) [inline compute/elastic-metal/how-to/index.mdx:103]:
```suggestion
3. Save the file and exit the text editor.
```

File: compute/elastic-metal/how-to/enable-smtp.mdx [hunk 13]
```diff
@@ -0,0 +1,54 @@
...
+content:
+  h1: How to enable STMP for Elastic Metal servers
+  paragraph: This page explains how to enable STMP for Elastic Metal servers
+dates:
+  validation: 2022-05-04
+  posted: 2022-05-04
+---
+
+By default, the SMTP ports (25, 465, 587) of your Elastic Metal server are blocked, and cannot be reached from our infrastructure. 
+If you want to send e-mails from your server, you can enable SMTP for each Availability Zone from the console.
```
Comment (nedaehm) [inline compute/elastic-metal/how-to/enable-smtp.mdx:14]:
```suggestion
If you want to send emails from your server, you can enable SMTP for each Availability Zone from the console.
```

File: compute/elastic-metal/how-to/index.mdx [hunk 14]
```diff
@@ -1,8 +1,141 @@
 ---
 meta:
-  title: Elastic Metal servers - How Tos
-  description: Elastic Metal How Tos
+  title: Setting up Gitlab with a Managed Database for PostgreSQL
```
Comment (RoRoJ) [inline compute/elastic-metal/how-to/index.mdx:3]: Seems like something has gone wrong here? I think this is the wrong content for this file.

Comment (bene2k1) [inline compute/elastic-metal/how-to/index.mdx:3]:
```suggestion
  title: Elastic Metal How Tos
```

**Pull request closed without merging**